webroot/js/components/player.js

```diff
@@ -95,6 +100,10 @@ class OwncastPlayer {
     }
   }
 
+  handleVolume(e) {
+    localStorage.setItem('owncastVolume', this.vjsPlayer.volume());
```

Comment:
There should be a `getLocalStorage`/`setLocalStorage` in `helpers.js`, could we use those instead so these getters and setters are centralized?

Reply:
Working on it!

Reply:
Done!